lasio/reader.py

```diff
@@ -449,7 +449,7 @@ def read_file_contents(file_obj, regexp_subs, value_null_subs, ignore_data=False
     return sections
 
 
-def inspect_data_section(file_obj, line_nos):
+def inspect_data_section(file_obj, line_nos, regexp_subs):
```

Comment:
Please add `regexp_subs` to the docstring

Reply:
Good catch,  working on it...